OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -185,8 +185,9 @@ public override Activity Tick(Actor self)
 					mobile.Facing,
 					mobile.Facing,
 					0);
+				ChildActivity = move;
```

Comment:
You can assign directly to `ChildActivity` here, no need for the intermediate variable.

There's another line further up  that would need to change. Unfortunately I can't put a comment there.
167: `return ActivityUtils.SequenceActivities(new Turn(self, firstFacing), this);`

Reusing activities like that is a bug. It would be better to queue the `Turn` as a child instead and then run that.